centralserver/i18n/management/commands/update_language_packs.py

```diff
@@ -1,858 +0,0 @@
-"""
```

Comment:
Looks like we still run this on the server?

https://github.com/fle-internal/ansible-playbooks/blob/master/roles/keep/tasks/main.yml#L245

https://github.com/fle-internal/ansible-playbooks/blob/master/roles/keep/templates/ka-lite-lang-pack-builder.sh.j2